hyp3_gamma/dem.py

```diff
@@ -0,0 +1,126 @@
+import json
+from pathlib import Path
+from subprocess import PIPE, run
+from tempfile import TemporaryDirectory
+from typing import Generator, List
+
+from hyp3lib import DemError
+from osgeo import gdal, ogr
+
+DEM_GEOJSON = '/vsicurl/https://asf-dem-west.s3.amazonaws.com/v2/cop30.geojson'
+
+gdal.UseExceptions()
+ogr.UseExceptions()
+
+
+def get_geometry_from_kml(kml_file: str) -> ogr.Geometry:
+    response = run(['ogr2ogr', '-wrapdateline', '-datelineoffset', '20', '-f', 'GeoJSON', '-mapfieldtype',
+                    'DateTime=String', '/vsistdout', kml_file], stdout=PIPE, check=True)
+    geojson = json.loads(response.stdout)
```

Comment:
I'm not sure I like `response` as the variable name holding a `CompletedProcess` object, and I think I'd prefer to keep the subprocess namespace since `run` is pretty generic.  

This might look better like:
```python
    ogr2ogr_cmd = ['ogr2ogr', '-wrapdateline', '-datelineoffset', '20', '-f', 'GeoJSON', '-mapfieldtype', 'DateTime=String', '/vsistdout', kml_file]
    geojson = run(ogr2ogr, stdout=PIPE, check=True).stdout
    geometry = json.loads(geojson)['features'][0]['geometry']
return ogr.CreateGeometryFromJson(json.dumps(geometry))